gcloud-java-compute/src/main/java/com/google/gcloud/compute/Compute.java

```diff
@@ -1586,6 +1634,110 @@ public static ImageListOption fields(ImageField... fields) {
   }
 
   /**
+   * Class for specifying disk get options.
+   */
+  class DiskOption extends Option {
+
+    private static final long serialVersionUID = -4354796876226661667L;
+
+    private DiskOption(ComputeRpc.Option option, Object value) {
+      super(option, value);
+    }
+
+    /**
+     * Returns an option to specify the disk's fields to be returned by the RPC call. If this option
+     * is not provided, all disk's fields are returned. {@code DiskOption.fields} can be used to
+     * specify only the fields of interest. {@link Disk#diskId()},
+     * {@link DiskConfiguration#diskType()} and {@link SnapshotDiskConfiguration#sourceSnapshot()}
```

Comment:
Nit: could add "either" after "and" to resolve a tiny bit of grammatical ambiguity.